syncstorage-web/src/metrics.rs

```diff
@@ -0,0 +1,30 @@
+use actix_web::{dev::Payload, web::Data, FromRequest, HttpRequest};
+use futures::future;
+use futures::future::Ready;
+use syncserver_common::{Metrics, Taggable};
+
+use super::ServerState;
+
+pub struct MetricsWrapper(pub Metrics);
```

Comment:
This serves the same function as `TokenserverMetrics` (it's a newtype that lets us define `FromRequest` on `Metrics`). We should probably be consistent and:
1. Rename it to `SyncstorageMetrics`; and
2. Move it to the extractors file